edn/src/edn.rustpeg

```diff
@@ -80,14 +82,22 @@ symbol -> Value = s:$( symbol_char_initial symbol_char_subsequent* ) {
     Value::Symbol(s.to_string())
 }
 
+namespace_divider = "."
+namespace_separator = "/"
 keyword_char_initial = ":"
+
 // TODO: More chars here?
```

Comment:
Is this comment still relevant?  `_`, `*`, and `-` seem like good name{space} chars...